templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -394,12 +394,12 @@ Parameters:
     Default: 200
     Description: Database allocated storage size, in gigabytes (GB). If you choose Provisioned IOPS, storage should be between 100 and 6144 GB.
     Type: Number
-  DBStorageEncrypted:
-    Default: "false"
+  EncryptionAtRest:
+    Default: "true"
```

Comment:
Assuming this default stance is (doesn't break) okay for existing stacks that are not encrypted?